.azure-pipelines.yml

```diff
@@ -269,6 +269,53 @@ jobs:
 #################################################################################
 
 
+- job: 'linux_pypi_build'
+  displayName: 'Linux PyPI build'
+  condition: startsWith(variables['build.sourceBranch'], 'refs/tags/')
+  timeoutInMinutes: 60
+  pool:
+    vmImage: 'ubuntu-latest'
+  steps:
+  - checkout: self
+    submodules: true
+  - bash: |
+      set -eux
+      sudo apt-get install -y python python-pip xvfb xorg
+      pip install twine
+    displayName: 'Bootstrap'
+  - bash: |
+      set -eux
+      python setup.py sdist bdist_wheel
+  - bash: |
+      set -eux
+      pip install --user virtualenv
+      python -m venv venv
+      source venv/bin/activate
+      pip install dist/*.whl[all]
+      xvfb run parsec core gui
+      deactivate
+  - bash: |
+      source venv/bin/activate
+      (sleep 3; killall xvfb-run) & xvfb-run parsec core gui; test $? -eq 143 || exit -1
```

Comment:
what's the sleep&killall for ? (we should add a comment about this)

also it would be better to add the `set -eux` at the begining of this script to know what was executed in case of error

Reply:
Otherwise we would have the GUI staying open forever... We just want to check we can launch it, kill it, and get the correct error code. A better test would simulate clicks to check the window do appear on screen, then quit the app. We should probably also use 

We can't have set -eux as the killing of the running xvfb would appear as an error. But we can have an exit -1 in case we can't obtain the expected error code.

Reply:
you should be able to do `set -eu` to keep comand info without stopping on crash
Also you should add a comment about why we need this

Also you should divide this line into multiples (I only realized now that it was `(sleep 3; killall xvfb-run) & xvfb-run [...]` and not `(sleep 3; killall xvfb-run) && xvfb-run [...]` xD)

Reply:
:crying_cat_face: